docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+services:
```

Comment:
I believe there are too many redundant services here, perhaps you forgot to delete some after copying
@nvthiet1995